### PR TITLE
Remove documentation around cookies

### PR DESCRIPTION
**Pull request by JaKraken**

### CODE REVIEW

File: docs/data-storage.md [hunk 1]
```diff
@@ -1,80 +1,38 @@
...
-
-*All games must be prepared to handle this scenario.*
+# Using Local Storage
 
 ## Saving data
 
-````
-gmi.setGameData(key, value);
-````
+Ideally, data should not be stored. If it is stored, data must not be personal and should adhere to the cookie policy of the browser. If the game contains sign in, it must respect the privacy settings on the user's account.
```
Comment (binhbui) [inline docs/data-storage.md]: Can get rid of this bit now.

File: docs/data-storage.md [hunk 2]
```diff
@@ -1,80 +1,38 @@
...
+[Global settings](settings.md#global-settings) can be accessed like so:
 
-[Manage Cookie Settings (TEST)]: http://www.test.bbc.co.uk/privacy/cookies/managing/cookie-settings
-[Manage Cookie Settings (LIVE)]: http://www.bbc.co.uk/privacy/cookies/managing/cookie-settings
+```javascript
+gmi.getAllSettings().audio;
+gmi.getAllSettings().motion;
+gmi.getAllSettings().subtitles;
 
-* [Home](../README.md)
```
Comment (binhbui) [inline docs/data-storage.md]: Did you mean to delete this?

File: docs/data-storage.md (hunk 1 above)
Comment (JaKraken) [reply docs/data-storage.md]: I was asked to keep this section with some changes in:
https://github.com/bbc/genie-starter-pack/pull/4

File: docs/data-storage.md (hunk 2 above)
Comment (JaKraken) [reply docs/data-storage.md]: This was just because i tried to use the same document across all our repos for consistency ill add the links back in

**Comment by Leonie-** [comment]
We will also need to update `index.html` to the fake GMI reflects the changes in the real one i.e. remove all `areCookiesAllowed` logic). Give me a shout if it's not clear.

**Pull request merged**